### PR TITLE
add jwt interceptor

### PR DESCRIPTION
**Pull request by SlakvichAleksandr**

### CODE REVIEW

File: frontend/src/app/core/interceptors/jwt.interceptor.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+@Injectable()
+export class JwtInterceptor implements HttpInterceptor {
+    // eslint-disable-next-line no-empty-function
+    constructor(private authService: AuthService) {}
+
+    public intercept(req: HttpRequest<any>, next: HttpHandler): Observable<HttpEvent<any>> {
+        return from(this.handleAccess(req, next));
+    }
+
+    private async handleAccess(req: HttpRequest<any>, next: HttpHandler): Promise<HttpEvent<any>> {
```
Comment (tatianahutii) [inline frontend/src/app/core/interceptors/jwt.interceptor.ts]: Please do not use any, you can just remove output type

**Review by tatianahutii — CHANGES_REQUESTED** [review]

File: frontend/src/app/core/interceptors/jwt.interceptor.ts (hunk 1 above)
Comment (SlakvichAleksandr) [reply frontend/src/app/core/interceptors/jwt.interceptor.ts]: what about input parameter?

Comment (tatianahutii) [reply frontend/src/app/core/interceptors/jwt.interceptor.ts]: Yes, we need to get rid of any in input as well

Comment (SlakvichAleksandr) [reply frontend/src/app/core/interceptors/jwt.interceptor.ts]: Can we use object for example instead of any in input types?

Comment (tatianahutii) [reply frontend/src/app/core/interceptors/jwt.interceptor.ts]: You can use unknown

**Review by tatianahutii — APPROVED** [review]
LGTM

**Pull request merged**